Content.Server/Nutrition/EntitySystems/ThirstSystem.cs

```diff
@@ -43,9 +38,6 @@ private void OnComponentStartup(EntityUid uid, ThirstComponent component, Compon
 
         private void OnRefreshMovespeed(EntityUid uid, ThirstComponent component, RefreshMovementSpeedModifiersEvent args)
         {
-            if (_jetpack.IsUserFlying(component.Owner))
```

Comment:
what does this have to do with traits

Reply:
Nothing, but I came across it and it's not supposed to be there. I could make it a separate PR if you want.

Reply:
Is there corresponding code to fix the mispredict?

Reply:
Undid the changes for now, and added a TODO